src/components/ShareButton/ShareButton.tsx

```diff
@@ -56,8 +63,16 @@ export function ShareButton({
     },
   ];
 
+  if (collect) {
```

Comment:
I guess we can check if the user has access to collections with the feature flag here to prevent from the button from showing up 😬 
```suggestion
  if (collect && features.collections) {
```